tests/cli/plugins/audit_plugin/test_audit_records.py

```
@@ -10,8 +10,8 @@
     ("input_dict", "expected_result"),
     [
         ({"a": 1, "b": 2}, {"a": 1, "b": 2}),  # simple dict
-        ({"a": {"b": 1, "c": {"d": 2}}}, {"a.b": 1, "a.c.d": 2}),  # nested_dict
-        ({"a": [{"b": 1}, {"c": 2}]}, {"a[0].b": 1, "a[1].c": 2}),  # dict with a list
+        ({"a": {"b": 1, "c": {"d": 2}}}, {"a.b": 1, "a.c.d": 2}),  # noqa: WPS221
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS221

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)